op-batcher/batcher/driver.go

```diff
@@ -606,3 +679,50 @@ func logFields(xs ...any) (fs []any) {
 	}
 	return fs
 }
+
+// JamTxPool is for testing ONLY. It sends a txpool-blocking transaction in a new (non-blocking)
+// goroutine to the txpool that will panic if the transaction completes successfully. (Tests should
+// expect the blocking transaction to result in error from being cancelled.) This function must be
+// called *before* the batcher starts submitting batches to ensure successful jamming, and will
+// panic otherwise.
+func (l *BatchSubmitter) JamTxPool(ctx context.Context) {
```

Comment:
done

Reply:
done, though this identified a race condition where the batcher can start before the jamming goroutine starts executing.  Fixed for now by inserting a 1 second sleep to give it time to start up. Another solution could be to just use a short timeout so that the jamming function doesn't block for too long, allowing it to be invoked without its own goroutine without blocking the entire test, but I think that's just effectively doing the same thing as an explicit sleep.

Reply:
I can fix this by separating out the send from the waiting in txmgr... let me do that.

Reply:
this is now done